spec/tbd_tests_spec.rb

```diff
@@ -1205,7 +1288,6 @@
     translator = OpenStudio::OSVersion::VersionTranslator.new
     TBD.clean!
 
```

Comment:
This RSpec is pretty much redundant now, as preceding RSpec (KIVA, set.osm) operates with similar use cases. May remove before releasing next version.